python/mxnet/gluon/data/vision/transforms.py

```diff
@@ -228,6 +228,54 @@ def forward(self, x):
         return image.random_size_crop(x, *self._args)[0]
 
 
+class Crop(HybridBlock):
```

Comment:
there is an deprecated mx.sym.Crop op already, using `sym.image.Crop` may introduce some confusion again, can you propose a new name?

For image transformation, since resize is supported, I guess CropResize is better?

Reply:
Do you intend to expand the implementation to add more parameters like x1, y1 ? If so then it makes sense to keep them x0 and y0. Else I think you should change them to x and y

Reply:
thanks @access2rohit I assume now I only need one x and y so I'll go with x and y